### DISCUSSION TITLE
Is this an abandonware?

### DISCUSSION THREAD

**Discussion by urbantom** [Microsoft/Partner-Center-PowerShell#400] [General]
I see the last commit on Feb 2021.
Microsoft implemented a significant progress toward M$ Graph-based API over the last 1.5 years.
Does this module support that recent Microsoft approach?

**Comment by marcelbraak** [comment]
Yes, it's better to start using the new MS Graph instead. The Powershell module is lacking support for NCE and other stuff.

**Reply by urbantom** [reply]
Thank you for the honest answer. When you're saying 'it's better to start using the new MS Graph', do you mean a direct MS Graph REST calls or any Powershell module enveloping the details?

**Comment by jonwbstr** [comment]
@idwilliams
